src/main/java/com/alibaba/excel/write/metadata/holder/WriteWorkbookHolder.java

```diff
@@ -134,6 +134,8 @@ public WriteWorkbookHolder(WriteWorkbook writeWorkbook) {
         try {
             copyTemplate();
         } catch (IOException e) {
+            throw new IOException(e.getMessage(), e);
```

Comment:
why not just throw this as a general exception after ExcelGenerateException? Catching the same exception and throwing it seems pointless here.